cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -3,23 +3,38 @@
     :copyright: Copyright © 2014-2015 Cask Data, Inc.
 
 ============================================
-Runtime Arguments
+Preferences
 ============================================
 
 .. include:: ../../../_common/_include/include-v260-deprecate-procedures.rst
 
-Flows, Procedures, MapReduce programs, and Workflows can receive runtime arguments:
+Preferences provide the ability to save configuration at various levels of the system including CDAP Instance,
+Namespace, Application and Program. Configuration is represented by a map of string-string pairs. Preferences
+can be retrieved/saved/deleted through REST API and/or through CLI. When programs are started, the Preferences
+at different levels are collapsed into one map and it is accessible through ways described later. Preferences
+are persisted across restart of Programs.
 
-- For Flows and Procedures, runtime arguments are available to the ``initialize`` method in the context.
+The overriding order of Preferences is:
 
-- For MapReduce, runtime arguments are available to the ``beforeSubmit`` and ``onFinish`` methods in the context.
+CDAP Instance < Namespace < Application < Program < Runtime Arguments (passed in during start of the Program)
+
+For example, if a configuration say, ``SAMPLE_KEY`` is set to 10 at Namespace level and at the Program level,
+it is set to 5, then when the Program is started, the value set at Program level will override the value set at
+Namespace level and thus will get 5 as the value for the key ``SAMPLE_KEY``.
+
+Programs such as Flows, Procedures, MapReduce programs, Services, Workflows etc will receive the resolved preferences
```

Comment:
Let's drop mention of Procedures in any new copy:

Programs such as Flows, MapReduce, Spark, Services, Workflows, etc. will receive the resolved preferences and they can be accessed through the ``getRuntimeArguments`` method of the context:

``getRuntimeArguments``: wouldn't a better name be available? You're really accessing the resolved preferences...